x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/data/ConstantNullBlock.java

```diff
@@ -83,6 +83,11 @@ public ConstantNullBlock filter(int... positions) {
         return (ConstantNullBlock) blockFactory().newConstantNullBlock(positions.length);
     }
 
+    @Override
+    public ConstantNullBlock keepMask(BooleanVector mask) {
+        return (ConstantNullBlock) blockFactory().newConstantNullBlock(getPositionCount());
```

Comment:
Shouldn't we just incref?

Reply:
I think I'll have a look at this one on it's own - there are a few cases where I think we can `incRef; return this;` in this class that we're not doing.